security/fma-operator-fee.md

```diff
@@ -126,13 +167,29 @@ Below are references for this project:
 
 Below is what needs to be done before launch to reduce the chances of the above failure modes occurring, and to ensure they can be detected and recovered from:
 
+ 
 - [ ] Coordinate with wallet providers to update their fee estimation logic
 - [ ] Implement automated monitoring on dabase growth rate
 
+**Testing**
```

Comment:
@teddyknox, @yoonchung8822  - If we need to resolve all of the newly added and mentioned AIs for the Operator fee FMA, does that mean we will need to move this feature for some next upgrade, since the general rule we followed in the past is that we need to close all AI before we ship a feature to Mainnet.